core/flyout.js

```diff
@@ -774,6 +774,7 @@ Blockly.Flyout.prototype.onMouseDown_ = function(e) {
   this.dragMode_ = Blockly.DRAG_FREE;
   this.startDragMouseY_ = e.clientY;
   this.startDragMouseX_ = e.clientX;
+  Blockly.Flyout.startFlyout_ = this;
```

Comment:
You shouldn't need to use Blockly.Flyout.startFlyout_.  For a given workspace there's only one flyout, so just use that.